### ISSUE TITLE
Event parser throws ArgumentOutOfRangeException when repeat flag is passed without an argument

### ISSUE THREAD

**Issue by pyrox18** [pyrox18/SchedulerBot#8]
Need to add checks for repeat flag arguments (and potentially all other flags) to prevent this error from occurring.

https://sentry.io/pyrox18/schedulerbot/issues/716327494/

```
ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
  File "/src/SchedulerBot.Client/Parsers/EventParser.cs", line 85, in ParseEventInputFlags
    Void ParseEventInputFlags(System.String[], SchedulerBot.Data.Models.Event ByRef, System.String)
  File "/src/SchedulerBot.Client/Parsers/EventParser.cs", line 29, in ParseNewEvent
    SchedulerBot.Data.Models.Event ParseNewEvent(System.String[], System.String)
  File "/src/SchedulerBot.Client/Commands/EventCommands.cs", line 53, in Create
    Void MoveNext()
  Module "DSharpPlus.CommandsNext.Command", line 654, in ExecuteAsync
    Void MoveNext()
...
(1 additional frame(s) were not displayed)

Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
```

**Comment by jespirit** [comment]
How do I access the Sentry data? I created an account on Sentry.io.

**Comment by pyrox18** [comment]
Only I can access the Sentry project that tracks errors for my SchedulerBot deployment, since there can be some sensitive data that is essential for error reporting but shouldn't be exposed to anyone else. No need to worry about this issue though; I've fixed it in 9f63d33 on the dev branch and the issue will close itself once I merge a release into master.
